internal/providers/terraform/azure/testdata/container_registry_test/container_registry_test.golden

```diff
@@ -0,0 +1,33 @@
+
+ Name                                           Monthly Qty  Unit            Monthly Cost 
+                                                                                          
+ azurerm_container_registry.basic                                                         
+ ├─ Registry usage (Basic)                               30  days                   $5.00 
+ ├─ Storage (over 10 GB)                                140  GB-months             $14.00 
+ └─ Build vCPU                                      540,000  seconds               $54.00 
+                                                                                          
+ azurerm_container_registry.my_registry                                                   
+ ├─ Geo replication (1 location)                         30  days                  $50.00 
+ └─ Build vCPU                                      540,000  seconds               $54.00 
```

Comment:
The "Registry usage" cost component should always be applied, see https://azure.microsoft.com/en-us/pricing/details/container-registry/#pricing